policyengine_us/variables/gov/states/az/tax/income/credits/charitable_contribution/az_charitable_contributions_credit.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class az_charitable_contributions_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona charitable contributions credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+    reference = (
+        "https://law.justia.com/codes/arizona/2022/title-43/section-43-1088/"
+    )
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.credits.charitable_contribution.ceiling
+        filing_status = tax_unit("filing_status", period)
+        foster_care = add(
+            tax_unit,
+            period,
+            ["foster_care_charitable_organization"],
+        )
+        charitable_contributions = add(
```

Comment:
This is based only on charitable contributions to qualifying organizations as defined in [§ 43-1088(M)(4)](https://law.justia.com/codes/arizona/2022/title-43/section-43-1088/). Make a new variable for that similar to my suggestion for the foster care one.